### ISSUE TITLE
Set server settings when mod loaded

### ISSUE THREAD

**Issue by kaloczikvn** [kaloczikvn/kPM#11]
```
vars.miniMapSpotting true
vars.3dSpotting false
vars.killCam false
vars.nameTag false
vars.3pCam false
vars.friendlyFire true
vars.idleTimeout 0
vars.autoBalance false
vars.teamKillCountForKick 0
vars.teamKillKickForBan 0
vars.regenerateHealth false
vu.ColorCorrectionEnabled false
vu.SunFlareEnabled false
vu.SuppressionMultiplier 0
vu.DestructionEnabled false
```

**Comment by kaloczikvn** [comment]
https://github.com/kiwidoggie/VEXT-HVT/blob/master/ext/server/__init__.lua#L36

**Comment by kaloczikvn** [comment]
Fixed in ff126e1fd9b0e5d1757056f712eb9a27401b7a5a
